### DISCUSSION TITLE
How to change the default number of epochs when using EfficentAD?

### DISCUSSION THREAD

**Discussion by tamamoto0821** [open-edge-platform/anomalib#2294] [Q&A] [answered]
### What is the motivation for this task?

The number of epoch is 999 when using EfficentAD
I want to change this.

### Describe the solution you'd like

The number of epoch is 999 when using EfficentAD
I want to change this.

### Additional context

The number of epoch is 999 when using EfficentAD
I want to change this.

**Comment by samet-akcay** [comment, marked as answer]
You could set `max_epochs` argument from the `Engine`. 

Depending on whether you are using CLI or API, you could achieve this by the following:

# CLI
```bash
anomalib train --model anomalib.models.EfficientAd --data MVTec --trainer.max_epochs 3
```

# API
```python
from anomalib.data import MVTec
from anomalib.models import EfficientAd
from anomalib.engine import Engine

datamodule = MVTec()
model = EfficientAd()
engine = Engine(max_epochs=10)

engine.train(datamodule=datamodule, model=model)
```
